anker/tests/tests/amm.rs

```diff
@@ -0,0 +1,34 @@
+use lido::token::Lamports;
+use solana_program::program_pack::Pack;
+use solana_program_test::tokio;
+use solana_sdk::signer::Signer;
+use testlib::anker_context::Context;
+
+#[tokio::test]
+async fn test_successful_token_swap() {
+    let mut context = Context::new_with_initialized_token_pool().await;
+    let (st_sol_kp, st_sol_token) = context
```

Comment:
Nit: please write `keypair` instead of abbreviating it to keep the codebase readable for newcomers who may not be intimately familiar with the terminology.